.github/workflows/validate-vector.yaml

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: © 2024 Kevin Lu
+# SPDX-Licence-Identifier: AGPL-3.0-or-later
+name: Validate vector.json
+on:
+  push:
+    branches: [master]
+    paths:
+      - .github/workflows/validate-vector.yaml
+      - src/validate.py
+      - data/**
```

Comment:
Can you add additional triggers on
- pull_request to master changing the same files, so we can validate this workflow within this pull request
- workflow_dispatch